ci/scripts/jenkins_build.sh

```diff
@@ -0,0 +1,134 @@
+
+gpuci_logger "Check versions"
+python3 --version
+gcc --version
+g++ --version
+
+gpuci_logger "Check conda environment"
+conda info
+conda config --show-sources
+conda list --show-channel-urls
+
+gpuci_logger "Checking S3 cuDF cache"
+CONDA_BLD_DIR=/opt/conda/conda-bld
+CUDF_CONDA_COMMIT=$(git log -n 1 --pretty=format:%H -- ci/conda)
+CUDF_CONDA_CACHE_PATH="/cudf/${CUDA_VER}/${PYTHON_VER}/${RAPIDS_VER}/${CUDF_CONDA_COMMIT}/${NVARCH}/cudf_conda.tar.bz"
+CUDF_CONDA_CACHE_URL="${S3_URL}${CUDF_CONDA_CACHE_PATH}"
+CUDF_CONDA_TAR="${WORKSPACE_TMP}/cudf_conda.tar.bz"
+
+echo "Checking ${DISPLAY_URL}${CUDF_CONDA_CACHE_PATH}"
+set +e
+aws s3 cp --no-progress ${CUDF_CONDA_CACHE_URL} ${CUDF_CONDA_TAR}
+CUDF_CACHE_CHECK=$?
+set -e
+
+if [[ "${CUDF_CACHE_CHECK}" != "0" ]]; then
+      gpuci_logger "Cache miss, Building cuDF"
+      mkdir -p ${CONDA_BLD_DIR}
+      # The --no-build-id bit is needed for sccache
+      CONDA_ARGS="--no-build-id --output-folder ${CONDA_BLD_DIR} --skip-existing --no-test" ${MORPHEUS_ROOT}/ci/conda/recipes/run_conda_build.sh libcudf cudf
+
+      gpuci_logger "sccache usage for cudf build:"
+      sccache --show-stats
+      ZS=$(sccache --zero-stats)
+
+      gpuci_logger "Archiving cuDF build"
+      cd $(dirname ${CONDA_BLD_DIR})
+      tar cfj ${CUDF_CONDA_TAR} $(basename ${CONDA_BLD_DIR})
+      cd -
+      aws s3 cp --no-progress ${CUDF_CONDA_TAR} ${CUDF_CONDA_CACHE_URL}
+else
+      gpuci_logger "Cache hit, using cached cuDF"
+      cd $(dirname ${CONDA_BLD_DIR})
+      tar xf ${CUDF_CONDA_TAR}
+      cd -
+fi
+
+gpuci_logger "Installing cuDF"
+mamba install -q -y -c file://${CONDA_BLD_DIR} -c nvidia -c rapidsai -c conda-forge libcudf cudf
+
+gpuci_logger "Installing other dependencies"
+mamba env update -q -n morpheus -f ./docker/conda/environments/cuda${CUDA_VER}_dev.yml
+
+gpuci_logger "Check cmake & ninja"
+cmake --version
+ninja --version
+
+gpuci_logger "Configuring cmake for Morpheus"
+cmake -B build -G Ninja \
+      -DCMAKE_MESSAGE_CONTEXT_SHOW=ON \
+      -DMORPHEUS_BUILD_BENCHMARKS=ON \
+      -DMORPHEUS_BUILD_EXAMPLES=ON \
+      -DMORPHEUS_BUILD_TESTS=ON \
+      -DMORPHEUS_USE_CONDA=ON \
+      -DMORPHEUS_PYTHON_INPLACE_BUILD=ON \
+      -DMORPHEUS_USE_CCACHE=OFF \
+      -DCMAKE_C_COMPILER_LAUNCHER=sccache \
+      -DCMAKE_CXX_COMPILER_LAUNCHER=sccache \
+      -DCMAKE_CUDA_COMPILER_LAUNCHER=sccache \
+      .
+
+gpuci_logger "Building Morpheus"
+cmake --build build -j --parallel ${PARALLEL_LEVEL}
+
+gpuci_logger "sccache usage for morpheus build:"
+sccache --show-stats
+
+gpuci_logger "Installing Morpheus"
+pip install -e ${MORPHEUS_ROOT}
+
+gpuci_logger "Archiving results"
+mamba pack --quiet --force --ignore-editable-packages --ignore-missing-files --n-threads ${PARALLEL_LEVEL} -n morpheus -o ${WORKSPACE_TMP}/conda_env.tar.gz
+tar cfj ${WORKSPACE_TMP}/workspace.tar.bz --exclude=".git" --exclude="models" --exclude=".cache" ./
+ls -lh ${WORKSPACE_TMP}/
```

Comment:
I'm not sure about this section. You are packing up the entire conda environment but you installed morpheus inplace. Is this why you needed to pack up the workspace too?